crates/fuel-core/src/schema/chain.rs

```diff
@@ -816,20 +814,18 @@ impl ChainInfo {
     }
 
     async fn latest_block(&self, ctx: &Context<'_>) -> async_graphql::Result<Block> {
-        let query: &ReadView = ctx.data_unchecked();
+        let query = ctx.read_view()?;
 
         let latest_block = query.latest_block()?.into();
         Ok(latest_block)
     }
 
     async fn da_height(&self, ctx: &Context<'_>) -> U64 {
-        let query: &ReadView = ctx.data_unchecked();
-
-        let height = query
-            .da_height()
-            .expect("The blockchain always should have genesis block");
+        let Ok(query) = ctx.read_view() else {
+            return 0.into();
+        };
```

Comment:
Is this change of behavior intentional? Is returning zero correct?

Reply:
If I change the output type to `Result`, it will change the schema, and it will be a breaking change.

Reply:
My question is: should we rather panic or return zero? But all things considered I think zero is fine here.